lightpath/path.py

```diff
@@ -234,13 +236,17 @@ def blocking_devices(self) -> List[Device]:
                 block.append(prev_device)
             # check inserted
             elif curr_state is DeviceState.Inserted:
-                # Ignore devices with low enough transmssion
+                current_transmission *= curr_status.transmission
```

Comment:
Do we need a check here for `curr_status.transmission` <= 1? Maybe an exception or a min(transmission, 1) clamp?